### ISSUE TITLE
Text blocks are not supported

### ISSUE THREAD

**Issue by h1alexbel** [volodya-lombrozo/jtcop#262]
I can't use text blocks in java 17 having this plugin on.

```java
final String cql = 
       """
       CREATE KEYSPACE test
       WITH REPLICATION = {
        'class' : 'NetworkTopologyStrategy',
        'datacenter1' : 1
        };
       """;
```

This expression will be failed by JTCOP with a following message:
```
Text Block Literals are not supported
```

Fix it, please

**Comment by volodya-lombrozo** [comment]
@h1alexbel Thank you for the report

**Comment by volodya-lombrozo** [comment]
@h1alexbel The bug should be fixed in the `1.1.1` version.

**Comment by h1alexbel** [comment]
@volodya-lombrozo thanks so much
